charts/common/values.schema.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://s3.mgmtbi.ch/json-schemas/helm/common/x.y.z/values.schema.json",
```

Comment:
I'm not sure if this reference is needed, linting works locally without this line

Reply:
This is probably not needed currently, see use cases:
https://json-schema.org/understanding-json-schema/structuring.html